CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Fixed organization bulk action button text.
+- Switched from `Remove from Org` to `Add to Org`
```

Comment:
This can be removed, I think line 10 is sufficient

Reply:
Got it